src/styles/styles.js

```diff
@@ -1379,7 +1379,7 @@ const styles = {
         fontFamily: fontFamily.GTA,
         fontSize: variables.fontSizeNormal,
         borderWidth: 0,
-        borderRadius: 0,
+        borderRadius: variables.componentBorderRadiusSmall,
```

Comment:
Also, this border radius is wrong. This also affects the composer. The composer should not have any border radius. 

On the other hand, it seems that the Comment editor can have top-left and bottom-left  border radius.

Reply:
Visibility not but there will be. Try to match with the parent. Does corner of parent lie over the input? And is that rounded? If yes round the input cornor and if not, don't round it.

Reply:
After `maginVertical: 6` this is how it will look, the corner won't overlap with the parent. So, we don't need the border radius.
<img width="899" alt="Screenshot 2022-07-14 at 1 10 18 AM" src="https://user-images.githubusercontent.com/83179295/178817806-b4ced45f-5c14-47ec-9c37-df2bef332797.png">